build/dependencies.props

```diff
@@ -9,7 +9,7 @@
     <InternalAspNetCoreSdkPackageVersion>2.2.0-preview2-20181011.2</InternalAspNetCoreSdkPackageVersion>
     <LibuvPackageVersion>1.10.0</LibuvPackageVersion>
     <MicrosoftAspNetCoreAllPackageVersion>2.2.0-preview3-35457</MicrosoftAspNetCoreAllPackageVersion>
-    <MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>2.2.0-preview3-35457</MicrosoftAspNetCoreBenchmarkRunnerSourcesPackageVersion>
```

Comment:
Why not just `run.ps1 upgrade deps`?

Reply:
Screw that thing that installs random dotnet versions on my box, and now I get to develop using a custom build and see warnings where I code on stuff ;)